server/app.js

```diff
@@ -0,0 +1,60 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
```

Comment:
do not use `var`, instead use `const`
[see here - section 2.1](https://github.com/airbnb/javascript)